lighthouse-core/test/audits/third-party-facades-test.js

```diff
@@ -0,0 +1,514 @@
+                blockingTime: 0,
+                firstEndTime: 47786.422034999996,
+                firstStartTime: 47786.323843,
+                mainThreadTime: 0,
+                transferSize: 145772,
+                url: 'https://f.vimeocdn.com/p/3.22.3/js/player.js',
+              },
+              {
+                blockingTime: 0,
+                firstEndTime: 47786.422311999995,
+                firstStartTime: 47786.324528,
+                mainThreadTime: 0,
+                transferSize: 17633,
+                url: 'https://f.vimeocdn.com/p/3.22.3/css/player.css',
+              },
+              {
+                blockingTime: 0,
+                firstEndTime: 47786.634061000004,
+                firstStartTime: 47786.606134,
+                mainThreadTime: 0,
+                transferSize: 9313,
+                url: 'https://i.vimeocdn.com/video/784397921.webp?mw=1200&mh=675&q=70',
+              },
+              {
+                blockingTime: 0,
+                firstEndTime: 47786.291588,
+                firstStartTime: 47786.074447,
+                mainThreadTime: 0,
+                transferSize: 8300,
+                url: 'https://player.vimeo.com/video/336812660',
+              },
+              {
+                blockingTime: 0,
+                firstEndTime: 47786.47053,
+                firstStartTime: 47786.325692,
+                mainThreadTime: 0,
+                transferSize: 1474,
+                url: 'https://f.vimeocdn.com/js_opt/modules/utils/vuid.min.js',
+              },
+              {
+                blockingTime: 0,
+                firstEndTime: 47786.417184000005,
+                firstStartTime: 47786.32147,
+                mainThreadTime: 0,
+                transferSize: 1075,
+                url: 'https://i.vimeocdn.com/video/784397921.jpg?mw=80&q=85',
+              },
+              {
+                blockingTime: 0,
+                firstEndTime: 47787.641538,
+                firstStartTime: 47786.499527,
+                mainThreadTime: 0,
+                transferSize: 818,
+                url: 'https://vimeo.com/ablincoln/vuid?pid=a88cdaf56540a693f597632ffeeaf6a38f56542a1600197631',
+              },
+              {
+                blockingTime: 0,
+                firstEndTime: 47786.70353599999,
+                firstStartTime: 47786.608785,
+                mainThreadTime: 0,
+                transferSize: 110,
+                url: 'https://fresnel.vimeocdn.com/add/player-stats?beacon=1&session-id=a88cdaf56540a693f597632ffeeaf6a38f56542a1600197631',
+              },
+              {
+                blockingTime: 0,
+                firstEndTime: 47786.06986,
+                firstStartTime: 47786.069794,
+                mainThreadTime: 0,
+                transferSize: 0,
+                url: 'http://player.vimeo.com/video/336812660',
+              },
+            ],
+            type: 'subitems',
+          },
+        },
+      ]
+    );
+  });
+});
```

Comment:
can you add some tests where blockingTime is not 0?

Reply:
Possibly not, but I will investigate. Iframe widgets like YouTube and Vimeo will always have a blocking time of 0. The live chat widgets like intercom which do have non-zero blocking time may not be free to use.